project-base/storefront/components/Layout/SimpleLayout/SimpleLayout.tsx

```diff
@@ -8,14 +7,12 @@ type SimpleLayoutProps = {
 
 export const SimpleLayout: FC<SimpleLayoutProps> = ({ heading, children, standardWidth }) => (
     <Webline>
-        <div className="text-center">
-            <Heading type="h1">{heading}</Heading>
-        </div>
+        <h1 className="mb-3 text-center">{heading}</h1>
 
         {standardWidth ? (
             children
         ) : (
-            <div className="mr-24 flex w-full justify-center">
+            <div className="flex w-full justify-center">
```

Comment:
Why were we able to remove this?

Reply:
Better question is how we were able to put this here :D It does anything just add additional margin on the right side which is then cut by overflow.

Reply:
😆 okay, lol. Thanks